app/views/application/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1><%= @application.name %></h1>
+
+<h3> Street Address: <%= @application.street_address %></h3>
+<h3> City: <%= @application.city %></h3>
+<h3> State: <%= @application.state %></h3>
+<h3> Zip Code: <%= @application.zip_code %></h3>
+<h3> Description why you should have a dog: <%= @application.description %></h3>
+<h3> Add a Pet to this Application </h3>
+<%= form_with url: "/applications/#{@application.id}", method: :get, local: true do |f| %>
```

Comment:
IF your form is acting funny but passing in tests, make sure you add `data: { turbo: false}`